recipe-server/client/control_new/less/ant-override.less

```diff
@@ -20,3 +20,10 @@
 .ant-menu-submenu-title {
   background: rgba(0, 0, 0, 0.025);
 }
+
+// ant Card's `noHovering` doesn't seem to do anything, so this class makes up for it.
```

Comment:
IMO, we should leave this and make removing it a piece of the Ant upgrade, then.

At the moment, this patch does what we need it to, given the version we're using. If a later patch is applied to update Ant, I don't see why the duties of that patch wouldn't be to also phase out things such as this. That way, the end user gets a consistent UI, both before and after the code changes.